DigiMeSDK/Core/DMEAuthorizationManager.m

```diff
@@ -17,76 +17,70 @@
 #import <StoreKit/StoreKit.h>
 
 static NSString * const kCARequestSessionKey = @"CARequestSessionKey";
-static NSString * const kCARequestRegisteredAppID = @"CARequestRegisteredAppID";
-static NSString * const kCARequest3dPartyAppName = @"CARequest3dPartyAppName";
 static NSString * const kCADigimeResponse = @"CADigimeResponse";
-static NSString * const kDMEClientScheme = @"digime-ca-master";
-static NSString * const kDMEClientSchemePrefix = @"digime-ca-";
-static NSInteger  const kDMEClientAppstoreID = 1234541790;
-static NSTimeInterval const kCATimerInterval = 0.5;
 
-@interface DMEAuthorizationManager() <SKStoreProductViewControllerDelegate>
+@interface DMEAuthorizationManager()
 
-@property (nonatomic, strong) NSString *appId;
-@property (nonatomic, strong) CASession *session;
-@property (nonatomic, strong) CASessionManager *sessionManager;
+@property (nonatomic, strong, readonly) CASession *session;
+@property (nonatomic, strong, readonly) CASessionManager *sessionManager;
 @property (nonatomic) BOOL authInProgress;
 @property (nonatomic, copy, nullable) AuthorizationCompletionBlock authCompletionBlock;
-@property (nonatomic, strong) SKStoreProductViewController *storeViewController;
 
 @end
 
 @implementation DMEAuthorizationManager
 
-#pragma mark - Initialization
+#pragma mark - Interfacee Conformance
 
-- (instancetype)init
+@synthesize interfacer = _interfacer;
+
+- (instancetype)initWithInterfacer:(DMEMercuryInterfacer *__weak)interfacer
```

Comment:
This initializer isn't used. We shoould probably mark `init` as unavailable for completeness and use this one.